peps/pep-0815.rst

```diff
@@ -0,0 +1,2281 @@
+PEP: 815
+Title: Wheel Variants - extending platform awareness
+Author: Jonathan DEKHTIAR <jonathan@dekhtiar.com>,
+        Michał Górny <mgorny@quansight.com>,
+        Konstantin Schütze <konstin@mailbox.org>,
+        Ralf Gommers <ralf.gommers@gmail.com>,
+        Andrey Talman <atalman@meta.com>,
+        Charlie Marsh <crmarsh416@gmail.com>,
```

Comment:
Nit: Do you mind using `charlie.r.marsh@gmail.com` (if not `charlie@astral.sh`)?